server.go

```diff
@@ -1265,7 +1266,12 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 		trInfo.tr.LazyLog(&fmtStringer{"Unknown service %v", []interface{}{service}}, true)
```

Comment:
This tracing output is still incomplete.

Suggestion: do the tracing call after constructing `errDesc`, and pass `errDesc` to it.

Reply:
Good call, done. Also switched it to LazyPrintf, since it is simpler and there's no sensitive data.